pkg/controllers/job/job_controller_actions.go

```diff
@@ -521,9 +504,9 @@ func (cc *Controller) calcPGMinResources(job *vkv1.Job) *v1.ResourceList {
 	return &minAvailableTasksRes
 }
 
-func (cc *Controller) initJobStatus(job *vkv1.Job) (*vkv1.Job, error) {
+func (cc *Controller) initJobStatus(job *vkv1.Job) (bool, error) {
 	if job.Status.State.Phase != "" {
-		return job, nil
+		return false, nil
```

Comment:
can we remove bool?

Reply:
the func initJobStatus in createJob need UpdateStatus, if remove bool, will go on UpdateStatus and return err